tools/flaky-test-reporter/main.go

```diff
@@ -126,7 +127,17 @@ func githubOperations(ghToken string, repoData []RepoData, dryrun bool) (map[str
 	return gih.processGithubIssues(repoData, dryrun)
 }
 
+func isWeekend(t time.Time) bool {
+	weekDay := t.Weekday()
+	return weekDay == 6 || weekDay == 0
```

Comment:
use `time.Sunday` and `time.Saturday`: https://godoc.org/time#Weekday

I don't think we're supposed to assume `iota` is zero...

Reply:
that's right. done